catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/util/UiTestUtils.java

```diff
@@ -90,4 +96,14 @@ public static boolean comparePixelRgbaArrays(byte[] firstArray, byte[] secondArr
 		}
 		return true;
 	}
+
+	public static void setLocale(Locale locale) {
```

Comment:
just no.
this is a very very specific function now used in only 2 tests, this has no business beeing put in a global (!) utility function.

when NOT changing the language via language picker in setting activity, it is sufficient to just call  SettingsActivity.updateLocale instead (as should be sufficient in the other test) since the shared preferences are update independently of the updateLocale call in SettingsActivity.

also i dislike importing and utilizing uitest methods in a "unit" test. if id have to change something, i would not expect a unit test beeing affected by anything i do in espresso tests...